### DISCUSSION TITLE
Is there a feature that display remote directories in a tree-like format on the left side window?

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1333] [Ideas]
Clicking the tree icon below, is there a feature that allows you to directly open a remote directory, and then display it as a tree on the left sidebar, so that when you click on a file in the tree, it will be displayed in the editor, allowing you to conveniently edit and debug remote files?

![image](https://github.com/user-attachments/assets/2c15eb54-c1bd-479d-b012-d828d9008238)

**Comment by pyscripter** [comment]
Currently not.  However there are Windows File Explorer extensions that can do that.   They should also work with the PyScripter File Explorer.
